lang/csharp/src/apache/test/Schema/ArraySchemaTests.cs

```diff
@@ -0,0 +1,37 @@
+using NUnit.Framework;
```

Comment:
You need to add the license header at the top of this new file. See ArraySchema.cs for example

Reply:
Fixed